sov-modules/sov-app-template/src/lib.rs

```diff
@@ -134,38 +137,39 @@ where
                 Ok(verified_tx) => verified_tx,
                 Err(e) => {
                     self.revert_and_slash(batch_workspace);
```

Comment:
This isn't a new change, but I'm not sure that we want to slash here. I think there are potentially non-malicious (and unpreventable) events where the same transaction is included twice in the same slot by different sequencers - and the second one will fail stateful verification

Reply:
Removed the slashing and added TODO to check one more time in the future.